src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -0,0 +1,1507 @@
+        public static unsafe Vector<T> operator -(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                result.SetElement(index, ScalarSubtract(left.GetElement(index), right.GetElement(index)));
+            }
+
+            return result;
+        }
+
+        /// <summary>Multiplies two vectors together.</summary>
+        /// <param name="left">The first source vector.</param>
+        /// <param name="right">The second source vector.</param>
+        /// <returns>The product vector.</returns>
+        [Intrinsic]
+        public static unsafe Vector<T> operator *(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                result.SetElement(index, ScalarMultiply(left.GetElement(index), right.GetElement(index)));
+            }
+
+            return result;
+        }
+
+        /// <summary>Multiplies a vector by the given scalar.</summary>
+        /// <param name="value">The source vector.</param>
+        /// <param name="factor">The scalar value.</param>
+        /// <returns>The scaled vector.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector<T> operator *(Vector<T> value, T factor)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                result.SetElement(index, ScalarMultiply(value.GetElement(index), factor));
+            }
+
+            return result;
+        }
+
+
+        /// <summary>Multiplies a vector by the given scalar.</summary>
+        /// <param name="factor">The scalar value.</param>
+        /// <param name="value">The source vector.</param>
+        /// <returns>The scaled vector.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector<T> operator *(T factor, Vector<T> value) => value * factor;
+
+        /// <summary>Divides the first vector by the second.</summary>
+        /// <param name="left">The first source vector.</param>
+        /// <param name="right">The second source vector.</param>
+        /// <returns>The vector resulting from the division.</returns>
+        [Intrinsic]
+        public static unsafe Vector<T> operator /(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                result.SetElement(index, ScalarDivide(left.GetElement(index), right.GetElement(index)));
+            }
+
+            return result;
+        }
+
+        /// <summary>Negates a given vector.</summary>
+        /// <param name="value">The source vector.</param>
+        /// <returns>The negated vector.</returns>
+        public static Vector<T> operator -(Vector<T> value) => Zero - value;
```

Comment:
This one and `~` could likely benefit from the same optimization I did for `operator *(Vector<T>, T)` where it does a different `for` loop if not HWAccelerated.